src/custom/lib/hooks/useCurrencyLogoURIs/useCurrencyLogoURIsMod.ts

```diff
@@ -9,6 +9,7 @@ import XDaiLogo from 'assets/cow-swap/xdai.png'
 import { ADDRESS_IMAGE_OVERRIDE } from 'constants/tokens'
 import { NATIVE_CURRENCY_BUY_ADDRESS } from 'constants/index'
 import uriToHttp from 'lib/utils/uriToHttp'
+import { useTokenLogo } from '@src/custom/components/SearchModal/CurrencySearch/useTokenSearch'
```

Comment:
Coul this be a code smell to depend in a global hook on this `SearchModal/CurrencySearch` hook?